eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -386,39 +572,56 @@ def dynestyfitter(lc, model, meta, log, **kwargs):
     if nlive < min_nlive:
         log.writelog(f'**** WARNING: You should set run_nlive to at least {min_nlive} ****')
 
-    sampler = NestedSampler(ln_like, ptform, ndims,
+    if hasattr(meta, 'ncpu') and meta.ncpu > 1:
+        pool = Pool(meta.ncpu)
+        queue_size = meta.ncpu
+    else:
+        pool = None
+        queue_size = None
+    sampler = NestedSampler(ln_like, ptform, ndims, pool=pool, queue_size=queue_size,
                             bound=bound, sample=sample, nlive=nlive, logl_args = l_args,
                             ptform_args=[prior1, prior2, priortype])
     sampler.run_nested(dlogz=tol, print_progress=True)  # output progress bar
     res = sampler.results  # get results dictionary from sampler
+    if meta.ncpu > 1:
```

Comment:
Change to `if hasattr(meta, 'ncpu') and meta.ncpu > 1:` or define `meta.ncpu = 1` above (Line 263) when not given.